backend/mora/service/manager.py

```diff
@@ -13,20 +13,109 @@
 
 """
 import uuid
+import operator
+import flask
 
 from . import address
 from . import handlers
+from . import orgunit
+from . import facet
+from . import employee
 from .validation import validator
 from .. import common
 from .. import lora
 from .. import mapping
 from .. import util
 
 
-class ManagerRequestHandler(handlers.OrgFunkRequestHandler):
+class ManagerRequestHandler(
+    handlers.OrgFunkRequestHandler,
+    handlers.OrgFunkReadingRequestHandler
+):
+
+    __slots__ = ()
     role_type = 'manager'
     function_key = mapping.MANAGER_KEY
 
+    @classmethod
+    def finder(cls, c, type, objid):
+
+        found = super().finder(c, type, objid)
+        if not found and util.get_args_flag("inherit_manager"):
```

Comment:
This will break if we try to do `inherit_manager` while `type` is `e`, as your code expects it to always be `ou`
`inherit_manager` doesn't make sense for employees anyway, so add an additional check here to ensure we only do this for org units.

Reply:
Yes - returning an empty list when type e encountered.